tests/bstr/build.rs

```diff
@@ -1,7 +1,7 @@
 fn main() {
+    // The Windows crate manually injects various functions needed to implement BSTR.
+    // This test validates these are included.
     windows::build! {
-        // The Windows crate manually injects various functions needed to implement BSTR.
-        // This test validates these are included.
-        Windows::Win32::System::OleAutomation::BSTR
+        Windows::Win32::System::OleAutomation::BSTR,
     };
```

Comment:
Note that `rustfmt` removed these comments from:

```rust
use format_build_macro::{
    // The Windows crate manually injects various functions needed to implement BSTR.
    // This test validates these are included.
    Windows::Win32::System::OleAutomation::BSTR,
};
```
In order to omit the curly braces, instead of leaving the use path and comment within an extra pair (or move the comment above) :/